tests/e2e/workloads/app/cosbench/test_cosbench.py

```diff
@@ -21,6 +21,7 @@ def teardown():
 
 
 @workloads
+@ignore_leftovers
```

Comment:
any reason for ignoring the leftovers?

Reply:
In many local test runs I was facing this issue, cosbench pod was getting removed http://pastebin.test.redhat.com/1100744, any other way to resolve this issue other than adding ignore_leftovers ?? When i checked there are no resources left in the cluster/namespace.

Reply:
increasing the timeout value resolved this issue.